src/groups/mwc/mwcc/mwcc_multiqueuethreadpool.h

```diff
@@ -1262,7 +1262,7 @@ inline MultiQueueThreadPool<TYPE>::MultiQueueThreadPool(
                               bdlf::PlaceHolders::_1,
                               // arena
                               bdlf::PlaceHolders::_2),  // allocator
-         -1,
```

Comment:
I think it's better to set this param in the `config` argument (which is `MultiQueueThreadPoolConfig`), with the default value of `-1`. This way, we make sure that we don't break apps depending on `mwc`.

Reply:
yeah, done that